File: ambeth/Ambeth.Util/ambeth/collections/IListElem.cs
using System;

namespace De.Osthus.Ambeth.Collections
{
    public interface IListElem<V>
    {
        Object ListHandle { get; set; }

        IListElem<V> Prev { get; set; }

        IListElem<V> Next { get; set; }

        V ElemValue { get; set; }
    }
}

File: jambeth/jambeth-util/src/main/java/de/osthus/ambeth/collections/IListElem.java
package de.osthus.ambeth.collections;

public interface IListElem<V>
{
	Object getListHandle();

	void setListHandle(Object listHandle);

	IListElem<V> getPrev();

	void setPrev(IListElem<V> prev);

	IListElem<V> getNext();

	void setNext(IListElem<V> next);

	V getElemValue();

	void setElemValue(V elemValue);
}
